configs/terminate_http2_post.yaml

```diff
@@ -32,7 +32,8 @@ static_resources:
                   prefix: "/"
                   headers:
                   - name: ":method"
-                    exact_match: "POST"
+                    string_match:
```

Comment:
nit, doing a search in the code, still found some example need to update  for example:
https://github.com/envoyproxy/envoy/blob/main/docs/root/configuration/http/http_filters/tap_filter.rst
https://github.com/envoyproxy/envoy/blob/main/api/envoy/config/rbac/v3/rbac.proto#L63

Reply:
I was planning to only change those causing the test failure but it seems we'd better just change all of them.